message_processor.go

```diff
@@ -50,36 +96,40 @@ func consumeLog(lf *logPair) {
 			logrus.WithField("id", lf.info.ContainerID).WithError(err).Debug("Ignoring error")
 			dec = protoio.NewUint32DelimitedReader(lf.stream, binary.BigEndian, 1e6)
 		}
-		if sendMessage(lf.splunkl, &buf, lf.info.ContainerID) == false {
-			continue
-		}
-		if sendMessage(lf.jsonl, &buf, lf.info.ContainerID) == false {
-			continue
-		}
 
+		if shouldSendMessage(buf.Line) {
+			// Append to temp buffer
+			if tmpBuf.appendToTempBuffer(&buf) {
+				// Send message to splunk and json logger
+				sendMessage(lf.splunkl, &buf, tmpBuf, lf.info.ContainerID)
+				sendMessage(lf.jsonl, &buf, tmpBuf, lf.info.ContainerID)
+				//temp buffer and values reset
+				if tmpBuf.bufferReset {tmpBuf.resetTempBuffer()}
```

Comment:
This seems wired formatting ?

Reply:
to make things a little bit cleaner. we can merge `tmpBuf.bufferReset` condition check to `Reset` member function

Reply:
Changed the formatting. And moving condition to the function.